<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonalClearTools</class>
 <widget class="QMainWindow" name="PersonalClearTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>205</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>PersonalClearTools</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QCheckBox" name="vCbxDeleteFiles">
          <property name="text">
           <string>Delete Files</string>
          </property>
          <property name="iconSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QCheckBox" name="vCbxEmptyRecycleBin">
            <property name="text">
             <string>Empty Recycle Bin</string>
            </property>
            <property name="iconSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QProgressBar" name="progressBar">
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="vCbxClosePrograms">
        <property name="text">
         <string>Close Opened Programs</string>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="1" column="0">
             <widget class="QPushButton" name="vBtnCheckXMLFile">
              <property name="text">
               <string>Check XML File</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="1" column="0">
               <layout class="QGridLayout" name="gridLayout_12">
                <property name="sizeConstraint">
                 <enum>QLayout::SetDefaultConstraint</enum>
                </property>
                <property name="spacing">
                 <number>6</number>
                </property>
                <item row="1" column="0">
                 <layout class="QGridLayout" name="gridLayout_14"/>
                </item>
                <item row="0" column="0">
                 <widget class="QPushButton" name="vBtnRun">
                  <property name="text">
                   <string>RUN</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="vBtnCheckLogFiles">
              <property name="text">
               <string>Check Log Files</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="vStatusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
